sql/derived_tables/folio_reporting.po_instance.sql

```diff
@@ -0,0 +1,39 @@
+DROP TABLE IF EXISTS folio_reporting.po_instance;
+CREATE TABLE folio_reporting.po_instance AS
+
+SELECT 
+       po_purchase_orders.po_number AS po_number, 
+       organization_organizations.code AS vendor_code,
+       json_extract_path_text(po_purchase_orders.data, 'approved') AS status_approved,
+       json_extract_path_text(po_purchase_orders.data, 'metadata','createdDate') AS created_date,  
+       json_extract_path_text(po_purchase_orders.data, 'metadata','createdByUserId') AS created_by, 
+       json_extract_path_text(configuration_entries.value::json, 'name') AS created_location,
+       po_lines.instance_id AS pol_instance_id,

```

Comment:
It might be a good idea to use `json_extract_path_text()` for `po_lines.instance_id` if you have found that the attribute is present sometimes and not others.  I can make this change after merging this PR.